tests/btrfs_test.py

```diff
@@ -46,8 +46,8 @@ def test_plugin_version(self):
 
 class BtrfsTestCase(BtrfsTest):
 
-    def setUp(self):
-        self.dev_file = create_sparse_tempfile("btrfs_test", 1024**3)
+    def setUp(self, dev_file_size=1024**3):
```

Comment:
Please move this to a new separate function called from `setUp`, `setUp` is inherited from `unittest.TestCase` and I don't like overriding the parameters, we could easily overlook this when creating a new subclass of `BtrfsTestCase` and break things by having a custom `setUp`.

Reply:
Thanks, this is exactly what I had in mind.

I just don't like adding a new parameters to `setUp`, because it doesn't have any parameters in `unittest.TestCase` and it can cause confusion when adding new test cases in the future.